man/top_articles.Rd

```diff
@@ -4,22 +4,21 @@
 \alias{top_articles}
 \title{Retrieve Data on Top Articles}
 \usage{
-top_articles(project = "en.wikipedia", platform = "all", year = "2015",
-  month = "10", day = "01", reformat = TRUE, ...)
+top_articles(project = "en.wikipedia", platform = "all",
+  start = as.Date("2015-10-01"), granularity = "daily", reformat = TRUE,
```

Comment:
Lets the user use R date objects without worrying about parsing / conforming to a unique api. Also keeps parameters consistent across all functions.